parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -115,6 +135,27 @@ class ModalParkDetail extends React.Component {
                 {this.getSavedParkIcon()}
 
                 <div className="Details">
+                <div className="Section">
+                  <div className="Ratings">
+                    <span className="SectionTitle">Rating</span> <br/>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(1)}>
+                      <b>1</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating(2)}>
+                      <b>2</b>
+                    </button>
+                    <button id={"rating-1"} onClick={() => this.handleAddRating()}>
```

Comment:
shouldn't this have this.handleAddRating(3)?

Reply:
Also button ids are identical for them, is that intentional? If so is there a reason they all have 1 attached.

Reply:
yeah make sure that the ids are different!